pkg/apis/triggers/v1alpha1/event_listener_types.go

```diff
@@ -77,6 +77,12 @@ type EventInterceptor struct {
 	// interceptor request headers. This allows the interceptor to make
 	// decisions specific to an EventListenerTrigger.
 	Header []pipelinev1.Param `json:"header,omitempty"`
```

Comment:
Does the Header field only apply to Webhook interceptors? If so, probably should be within the `WebhookInterceptor` struct?

Reply:
Yeah I think so. I misunderstood what it did earlier.